test/e2e/kubectl/kubectl.go

```diff
@@ -1832,6 +1842,20 @@ metadata:
 	})
 })
 
+func findOcc(list []string, item string) []int {
+	occ := make([]int, 0)
+	for i, val := range list {
+		if strings.Contains(val, item) {
+			occ = append(occ, i)
+		}
+	}
+	return occ
+}
+
+func splitVersionString(r rune) bool {
```

Comment:
maybe docstring here too?

Reply:
I thought we add comments only to exported functions, will add